pages.tr/sunos/devfsadm.md

```diff
@@ -0,0 +1,16 @@
+# devfsadm
+
+> `/dev` için yönetim komutu. `/dev` ad alanına yönetir.
+> Daha fazla bilgi için: <https://www.unix.com/man-page/sunos/1m/devfsadm>.
```

Comment:
```suggestion
> Daha fazla bilgi: <https://www.unix.com/man-page/sunos/1m/devfsadm>.
```
Same doubt here.

Reply:
"Daha fazla bilgi için" is much more elegant. "Daha fazla bilgi" doesn't sound native but sounds like a direct English to Turkish translation done by a bot.

Reply:
But our standard translation is `Daha fazla bilgi` in https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py All the same, or change all Turkish pages

Reply:
All Turkish pages that lack "için" should have it. I will make a PR for that.

Reply:
> All Turkish pages that lack "için" should have it. I will make a PR for that.

Great, I will add in my approval.